src/watchers/html.ts

```diff
@@ -59,6 +60,71 @@ class HTMLWatcher extends DOMWatcher<Instance> {
 
       this._sendEvent(eavs);
     });
+
+    let {program:me} = this;
```

Comment:
For consistency, let's chain these calls instead of putting them into the `me variable.

E.g.

```
this.program
  .watch(...)
  .asObjects(...)
  .watch(...)
  // ...
```